### DISCUSSION TITLE
Calendar Widget  has poor UI when date required is a long time from now.

### DISCUSSION THREAD

**Discussion by artumi-richard** [community/community#40445] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi

Loving GitHub Projects, and I'm looking at implementing a "Hold Date" for tasks. The idea is that if you are blocked then the task would be "On Hold" until a certain time. I want users to have the ability to set the hold date in the future. 

And then I see that in order to set a hold date next year I need to do 12 clicks.

 
![image](https://user-images.githubusercontent.com/743415/204810718-e976cd49-1efd-4ca1-af1d-edaae43c318b.png)

I can't even type in a date, which I would have been happy with.  The browser HTML5 Date pickers are much better than they used to me. Perhaps it's time Github re-evaluated how to do this? 

It's completely unusable for columns like "Date of Birth" or "First publication date".

Is this the right place to bang the drum to get this changed?

**Comment by ossanna16** [comment]
Hi there @artumi-richard and welcome to our community! Thank you for the feedback, this is absolutely the right place!

**Reply by artumi-richard** [reply]
Hi @ossanna16 . That's great to hear, thanks. What should I expect next? Is there some kind of vote? Or are you the person who would raise this for some kind of internal github discussion?

**Reply by ossanna16** [reply]
Our product teams are keeping a close eye on Discussions and will add your feedback to their feature request list, which will then be discussed internally.

**Comment by ntap-bmegan** [comment]
Apologies if it's not appropriate to add a workaround here, but this is something that frustrated me. I noticed that if you click through to the issue and expand the project metadata there, you can type in the field:

![image](https://user-images.githubusercontent.com/56453501/216335274-7d34704a-a5ad-46de-80c5-09cff1ec212f.png)

Unfortunately you can't just paste a date, as each part of the date is its own segment.

**Reply by artumi-richard** [reply]
Thanks for finding the workaround, and putting it here. It will help me at least.
